Woof/Woof/View/DetailedPetSitterView/ContactDelailsSectionView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+/// A view displaying a section for the detailed sitter's information.
+struct ContactDelailsSectionView: View {
+    /// The name of an icon.
+    let image: String
+    /// The informational text to be displayed.
+    let message: String
+
+    var body: some View {
+        RoundedRectangle(cornerRadius: AppStyle.UIElementConstant.cornerRadius)
+            .padding(.horizontal)
+            .frame(height: AppStyle.UIElementConstant.frameContactDetailsSectionHeight)
+            .overlay(HStack {
+                Image(systemName: image)
+                Text(message)
+                    .foregroundColor(Color.App.grayDark)
+                    .fontWeight(.bold)
+            }.foregroundColor(Color.App.purpleDark))
```

Comment:
Maybe better to specify color directly for the image, not for the HStack, if you want to change color of the icon
```suggestion
                       .overlay(
                HStack {
                    Image(systemName: image)
                        .foregroundColor(Color.App.purpleDark)
                    Text(message)
                        .foregroundColor(Color.App.grayDark)
                        .fontWeight(.bold)
                }
            )
```

Reply:
Thanks, fixed it.